crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -184,38 +231,150 @@ impl<'w, 's> Commands<'w, 's> {
+    ///     high_score: 0,
+    /// });
+    /// # }
+    /// # system.system();
+    /// ```
     pub fn insert_resource<T: Component>(&mut self, resource: T) {
         self.queue.push(InsertResource { resource })
     }
 
-    /// Queue a resource removal.
+    /// Removes a resource from the [`World`].
+    ///
+    /// See [`World::remove_resource`] for more details.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use bevy_ecs::prelude::*;
+    /// #
+    /// # struct Scoreboard {
+    /// #     current_score: u32,
+    /// #     high_score: u32,
+    /// # }
+    /// #
+    /// # fn system(mut commands: Commands) {
+    /// commands.remove_resource::<Scoreboard>();
+    /// # }
+    /// # system.system();
+    /// ```
     pub fn remove_resource<T: Component>(&mut self) {
         self.queue.push(RemoveResource::<T> {
             phantom: PhantomData,
         });
     }
 
     /// Adds a command directly to the command list.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use bevy_ecs::prelude::*;
+    /// use bevy_ecs::system::InsertBundle;
+    /// #
+    /// # struct PlayerEntity { entity: Entity }
+    /// # struct Health(u32);
+    /// # struct Strength(u32);
+    /// # struct Defense(u32);
+    /// #
+    /// # #[derive(Bundle)]
+    /// # struct CombatBundle {
+    /// #     health: Health,
+    /// #     strength: Strength,
+    /// #     defense: Defense,
+    /// # }
+    ///
+    /// fn add_combat_stats_system(mut commands: Commands, player: Res<PlayerEntity>) {
+    ///     commands.add(InsertBundle {
+    ///         entity: player.entity,
+    ///         bundle: CombatBundle {
+    ///             health: Health(100),
+    ///             strength: Strength(40),
+    ///             defense: Defense(20),
+    ///         },
+    ///     });
+    /// }
+    /// # add_combat_stats_system.system();
+    /// ```
     pub fn add<C: Command>(&mut self, command: C) {
         self.queue.push(command);
     }
 }
 
-/// A list of commands that will be run to modify an [`Entity`].
+/// A list of commands that will be run to modify an [entity](crate::entity).
 pub struct EntityCommands<'w, 's, 'a> {
     entity: Entity,
     commands: &'a mut Commands<'w, 's>,
 }
 
 impl<'w, 's, 'a> EntityCommands<'w, 's, 'a> {
-    /// Retrieves the current entity's unique [`Entity`] id.
+    /// Returns the [`Entity`] id of the entity.
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use bevy_ecs::prelude::*;
+    /// #
+    /// fn my_system(mut commands: Commands) {
+    ///     let entity_id = commands.spawn().id();    
+    /// }
+    /// # my_system.system();
+    /// ```
     #[inline]
     pub fn id(&self) -> Entity {
         self.entity
     }
 
-    /// Adds a [`Bundle`] of components to the current entity.
+    /// Adds a [`Bundle`] of components to the entity.
+    ///
+    /// If you want to spawn a new entity, consider [`Commands::spawn_bundle`] for a more
+    /// condensed alternative. See
```

Comment:
I think its worth specifically calling out what it is an alternative _to_. spawn_bundle isn't an alternative to insert_bundle. It is an alternative to `spawn().insert_bundle()`.

Reply:
I removed that paragraph and added one for `spawn`.